### ISSUE TITLE
Error during Installation

### ISSUE THREAD

**Issue by pixelyo** [NeoTerrm/NeoTerm#10]
Hi,

I am getting this error during the installation process:

```
Opps!
android.system.ErrnoException: symlink failed: EEXIST (File exists)
```

any ideas why I get this error and how to solve it?

**Comment by licaon-kter** [comment]
Device?Android version?

**Comment by WordlessEcho** [comment]
Same here.
Nexus 5 running Android 8.0 (OPR1)

**Comment by pixelyo** [comment]
@licaon-kter sorry,

i am on ASUS ZenPad 8 (P024) with Android 7.1.2

**Comment by licaon-kter** [comment]
It's fresh install or update?

**Comment by pixelyo** [comment]
@licaon-kter it is a fresh install, the first one

**Comment by licaon-kter** [comment]
To version 1.2.10?

**Comment by pixelyo** [comment]
first I had only tried 1.2.8 from https://apt.izzysoft.de/fdroid/index/apk/io.neoterm

just now I tried 1.2.10 from https://github.com/NeoTerm/NeoTerm/releases and i get the same error message

**Comment by WordlessEcho** [comment]
```
11-13 10:50:59.751 E/NeoTerm-Setup(10987): [Thread:Thread-2], run(null:526): Linking libbz2.so.1.0.6 to /data/data/io.neoterm/files/usr-staging/lib/libbz2.so.1.0
11-13 10:50:59.760 E/NeoTerm-NeoTerm-Emulator(10987): [Thread:Thread-2], run(null:186): args[0] = Bootstrap error
11-13 10:50:59.760 E/NeoTerm-NeoTerm-Emulator(10987): args[1] = android.system.ErrnoException: symlink failed: EEXIST (File exists)
```

**Comment by licaon-kter** [comment]
State your setup!

**Comment by pixelyo** [comment]
@licaon-kter not sure what you all need,

Device: ASUS ZenPad 8 (P024)
Version: Android 7.1.2
Rom: LineageOS 14.1

**Comment by licaon-kter** [comment]
Not you ;)

**Comment by WordlessEcho** [comment]
Install from [CoolApk](https://www.coolapk.com/apk/io.neoterm), and open it.
Anyway, rooted by Magisk.

**Comment by imkiva** [comment]
Fixed in v2.0.0, check it out [here](https://github.com/NeoTerm/NeoTerm/releases/tag/v2.0.0)
